tapdb/assets_store.go

```diff
@@ -718,6 +742,17 @@ func dbAssetsToChainAssets(dbAssets []ConfirmedAsset,
 			AnchorMerkleRoot:       sprout.AnchorMerkleRoot,
 			AnchorTapscriptSibling: sprout.AnchorTapscriptSibling,
 		}
+
+		// We only set the lease info if the lease is actually still
+		// valid and hasn't expired.
+		owner := sprout.AnchorLeaseOwner
+		expiry := sprout.AnchorLeaseExpiry
+		if len(owner) > 0 && expiry.Valid &&
```

Comment:
How/when are leases to be cleaned up?

Reply:
This isn't currently done. Not sure what the best approach would be. In `btcwallet` I think we check the leases on certain actions. We could do the same here as well, e.g. whenever we list the coins or insert a new lease.
Or we could do it with a timed job. Going to see what makes most sense while addressing the other comments.